app/js/actions/actionTypes.js

```diff
@@ -1,2 +1,3 @@
 /*eslint-disable*/
```

Comment:
Please add reasons for disabling eslint on files, so we can know when to fix and enable linting back on those files.